test/integration/show_name_helper_test.rb

```diff
@@ -1,16 +1,25 @@
+# frozen_string_literal: true
+
 require "test_helper"
 
 # Tests of helper module
 class ShowNameHelperTest < IntegrationTestCase
```

Comment:
Is there a way to do this in a functional test? 
Note that (among other things) this tests for the presence of links in ShowObservation, where the links are supplied by ShowNameHelper.

Reply:
You're right.  It looks to me naively like you could just dump this almost verbatim into test/controllers/name_controller_test.rb and it would continue to work.